python/uyuni/common/rhnLib.py

```diff
@@ -129,7 +130,7 @@ def timestamp(s):
     timeval = list(time.strptime(s, format_string))
     # No daylight information available
     timeval[8] = -1
-    return time.mktime(tuple(timeval))
+    return calendar.timegm(tuple(timeval))
```

Comment:
This part is a bit confusing for me. I'm not fully sure if it's the best way to do it.
I don't like 3 points here:
1. New import of `calendar` just for this calculation
2. The logic of setting daylight information explicitly with `timeval[8] = -1`
3. Now it returns `int` instead of `float` as before

Maybe we can do it this way instead?:
```
return time.mktime(tuple(timeval))-time.timezone
```